lib/data_fetchers/git_fetcher.rb

```diff
@@ -38,18 +38,31 @@ def line_changes
   end
 
   def languages
-    languages = github.languages.map do |language, percent|
+    languages = github.languages
+      .merge(gitlab.languages) { |key, value1, value2| ((value1 + value2) / 2.0).round }
```

Comment:
You're right actually, summing up percentages doesn't work like this.
We could actually return the absolute numbers from github and gitlab (`Github#languages` and `GitlabClient#languages) and calculate the percentage here, then we'll have the correct average. Or am I missing something?